### ISSUE TITLE
Hyperparameter tuning for BERT

### ISSUE THREAD

**Issue by badaa002** [badaa002/DAT550-2025---Hyperpartisan-News-Detection-#15]
Try learning rate, batch size, epochs

Use validation scores to guide
